pages/1.11/tutorials/iot_pipeline/index.md

```diff
@@ -47,7 +47,7 @@ The following graphic illustrates the data flow:
 
 ## Prerequisites
 
-*  [DC/OS](/1.10/installing/oss/) or [DC/OS Enterprise](/1.10/installing/ent/) installed with at least 5 [private agents][6] and 1 [public agent][6].
+*  [DC/OS](/1.10/installing/) or [DC/OS Enterprise](/1.10/installing/ent/) installed with at least 5 [private agents][6] and 1 [public agent][6].
```

Comment:
You don't have to replace /1.10/ paths. Let it remain the way it is. 
/1.10/installing/oss/ and /1.10/installing/ent/ are correct.